src/stores/contactmoment/index.ts

```diff
@@ -106,6 +108,8 @@ export type ContactmomentContactVerzoek = {
 };
 
 export type ContactmomentKlant = {
+  bedrijfIdentifier?: BedrijfIdentifier;
```

Comment:
Heb je overwogen om hier simpelweg het bedrijf uit Kvk of de persoon uit Brp in een property op te nemen? Dan hoef je hier geen interne ids te genereren. Betekent wel dat de url van de detailpaginas anders wordt.

Reply:
besproken: de detailpagina zoekt de klant op uit de applicatie store en geeft de benodigde gegevens daaruit door aan de componenten. bijvoorbeeld bsn aan de klantdetailcomponent om de gegevens uit het brp op te halen. in tegenstelling tot wat we nu doen, namelijk de id meegeven aan het component en het component in de store de klant te laten opzoeken.

de rest van het voorstel doen we nog niet. Het bijhouden van de actieve klant in de store is op dit moment niet niet betrouwbaar genoeg. er is een story aangemaakt om dat te verbeteren. de voorgetelde aanpassing kunnen we eventueel daar in meenemen.

Reply:
dit gebeurt maar op 1 tab per pagina (brp gegevens en kvk gegevens). verplaatsten naar detailpagina levert bij nader inzien niets op. maakt het eerder rommeliger

Reply:
Prima. Ik denk dat het bij de story die je hebt aangemaakt alsnog nuttig is om te doen, maar dat zien we tegen die tijd wel